JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -103,6 +106,12 @@ fun AppDrawer(
     }
 }
 
+private fun dividerColor(isLight: Boolean) = if (isLight) {
```

Comment:
Couldn't this be an alpha modification of a theme color like `onSurface`?

Reply:
I admit my ignorance on that, will check if I can do the same. I went with an approach of "keep the colour where it was already defined" but will see if I can use a theme colour instead.